Fabric.Authorization.API/Models/External/Identity/IdentityUserSearchResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Fabric.Authorization.API.Models.External.Identity
+{
+    public class IdentityUserSearchResponse
```

Comment:
I'm wondering if this class would be better in a different namespace, it might be confusing to future developers that this model is really a model of a response from another service since it is located in the API/Models overall hierarchy.

Reply:
@michaelvidal24, do you think we should move code related to external services into an entirely different project?

Reply:
@bsmithvt - that's not a bad idea, but wee could also just have a namespace/folder:
>Fabric.Authorization.API.RemoteServices

and each remote service could have its own namespace:
>Fabric.Authorization.API.RemoteServices.Identity

and have a models namespace within each.